packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/package.json

```diff
@@ -11,8 +11,10 @@
     "exports": {
       "./package.json": "./package.json",
       ".": "./src/index.ts",
+      "./api": "./src/api/index.ts",
       "./models": "./src/models/index.ts",
-      "./api": "./src/api/index.ts"
+      "./models/fhir": "./src/models/fhir/index.ts",
+      "./models/fhir/r4": "./src/models/fhir/r4/index.ts"
```

Comment:
I am curious if we could enable the sub-path api-view file in our smoke-test?

Reply:
we can't, the subpath api view only works in js sdk repo with dev-tool.